management/user.go

```diff
@@ -340,3 +368,20 @@ func (m *UserManager) Blocks(id string) ([]*UserBlock, error) {
 func (m *UserManager) Unblock(id string) error {
 	return m.delete(m.uri("user-blocks", id))
 }
+
+// GenerateRecoveryCode removes the current multi-factor authentication recovery code and generate a new one.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/post_recovery_code_regeneration
+func (m *UserManager) GenerateRecoveryCode(id string) (*UserRecoveryCode, error) {
```

Comment:
Should this be `RegenerateRecoveryCode(...)`? As I understand it, the action is to replace an existing recovery code with a new one.

Reply:
I think you are right, fixing